src/client/helpers/cookieClientHelper.ts

```diff
@@ -4,17 +4,28 @@ export const getUsernameAndToken = async (): Promise<{
   username: string;
   accessToken: string;
 }> => {
-  //PARSE TOKEN AND USERNAME FROM COOKIES
+  // PARSE TOKEN AND USERNAME FROM COOKIES
```

Comment:
Please add comment with function name, description, param, and returns to include code readability.